visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts

```diff
@@ -281,6 +282,9 @@ export const whatIfTabStyles: () => IProcessedStyleSet<IWhatIfTabStyles> = () =>
         panelPlaceholderWrapper: [FabricStyles.missingParameterPlaceholder, {
```

Comment:
:O I haven't seen this before `[FabricStyles.missingParameterPlaceholder`

Reply:
yeah, it was my first attempt at sharing styles~! its the way of the future!